### PR TITLE
Prometheus exporter reporter implementation

### PR DESCRIPTION
**Pull request by JoseAlvaradoCevallos**
This PR modifies the prometheus exporter to adapt it to the new features added to the performance reporter. It removes the code in charge of creating the graph and keeps the json generator, adding a new parameter in which the title and units of each query are passed.

### CODE REVIEW

File: prometheus-exporter/README.md [hunk 1]
```diff
@@ -5,18 +5,30 @@ This tool provides a new tool called Prometheus Exporter, which uses Prometheus,
...
 
     -u: the URL of the Prometheus server to query
-    -a: the authorization header to use when querying the Prometheus server
+    -a: the authorization header or
+    -x: user and password to use when querying the Prometheus server 
     -s: the start date of the query range in Unix timestamp format
     -e: the end date of the query range in Unix timestamp format
     -t: the step size of the query in seconds
     -q: the Prometheus query to run. This can be a single query or multiple queries separated by semicolons.
+    -i: the title of these metrics and their units, separated by commas.
```
Comment (davidgca) [inline prometheus-exporter/README.md:29]: I suggest adding "and their units in the Y axis(X-axis should be always timestamp)"

Comment (JoseAlvaradoCevallos) [reply prometheus-exporter/README.md]: I like this suggestion. Added!

**Review by davidgca — APPROVED** [review]
LGTM

**Review by JavierUhagon — CHANGES_REQUESTED** [review]
Buen cambio y el código parece que está bastante bien, lo único lo que te he comentado del "estilo" del fichero, el código perfecto.

File: prometheus-exporter/script/prometheus_chart_exporter.py [hunk 2]
```diff
@@ -22,15 +22,31 @@ def url_type(arg_value, pat=re.compile(r"^https?://.+$")):
...
+    return values_list.split(';')
+
+def get_graph_info(graph_str):
+    graph_list = graph_str.split(",")
+    graph_dict = {
+        "title": graph_list[0],
+        "y_units": graph_list[1]
+    }
+    return graph_dict
+
```
[the remark this answers is not in the document]
Comment (JoseAlvaradoCevallos) [reply prometheus-exporter/script/prometheus_chart_exporter.py:40]: Okey, por ahora lo estandarizamos en 2 saltos lineas. Cambios realizados

**Pull request merged**